### PR TITLE
Adding PopUp Name

### PR DESCRIPTION
**Pull request by Rayane0823**

### CODE REVIEW

**Review by SebSamPro — APPROVED** [review]

**Review by SebSamPro — APPROVED** [review]

**Review by JulienOza — APPROVED** [review]

File: frontend/src/components/NamePopUp/NamePopUp.jsx
Comment (LucasMerino10) [inline frontend/src/components/NamePopUp/NamePopUp.jsx:1]: Divs :)

**Review by LucasMerino10 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Homepage/Homepage.css [hunk 1]
```diff
@@ -67,6 +60,10 @@ p {
   font-size: 1.1rem;
   margin-top: 0;
   padding-top: 0;
+  font-family: Knewave, sans-serif;
+  color: white;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Homepage/Homepage.css]: variable is better

File: frontend/src/components/NamePopUp/NamePopUp.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+.PopUpBackground {
+  background-color: #fff(200, 200, 200);
```
Comment (Yoann-Fortin) [inline frontend/src/components/NamePopUp/NamePopUp.css]: I need explanation

File: frontend/src/components/NamePopUp/NamePopUp.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import "./NamePopUp.css";
+
+function PopUp({ setOpeningPopUp }) {
+  return (
+    <div className="PopUpBackground">
```
Comment (Yoann-Fortin) [inline frontend/src/components/NamePopUp/NamePopUp.jsx]: Why a div here ?

File: frontend/src/components/NamePopUp/NamePopUp.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      <section className="PopUpContainer">
+        <p className="PopUpText">
+          Le principe consiste à deviner quel film se cache derrière l' affiche.
+          Ne t'en fais pas, des indices apparaissent au fur et à mesure du
+          chrono. Bonne chance !
+        </p>
+        <input className="PopUpInput" placeholder="Entrez votre nom" />
+        <div className="PopUpAllButtons">
```
Comment (Yoann-Fortin) [inline frontend/src/components/NamePopUp/NamePopUp.jsx]: another html tag is better

File: frontend/src/components/poster/Poster.jsx [hunk 5]
```diff
@@ -6,6 +6,7 @@ function Poster() {
       className="poster__img"
       src="https://i.ebayimg.com/images/g/HDwAAOSwFJZkvjiG/s-l1600.jpg"
       alt=""
+      draggable="false"
```
Comment (Yoann-Fortin) [inline frontend/src/components/poster/Poster.jsx:9]: Why draggable attribute ?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/poster/Poster.jsx (hunk 5 above)
Comment (Rayane0823) [reply frontend/src/components/poster/Poster.jsx:9]: Without this attribute, we can drag the image and the blur disappears. It is an anti-cheat.

**Review by LucasMerino10 — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**